relay-dynamic-config/src/feature.rs

```diff
@@ -1,7 +1,14 @@
 use std::collections::BTreeSet;
 
+use serde::ser::SerializeSeq;
 use serde::{Deserialize, Serialize};
 
+/// Feature flags that are not used by the current version of Relay, but have to be propagated
+/// to downstream Relays to operate correctly.
+///
+/// This is useful when a feature is supposed to be "always on" (after feature graduation).
+const HARDCODED_FEATURE_FLAGS: &[&str] = &["organizations:user-feedback-ingest"];
```

Comment:
Maybe we can just name it that way?

```suggestion
const GRADUATED_FEATURE_FLAGS: &[&str] = &["organizations:user-feedback-ingest"];
```